modules/Kconfig.mcuboot_bootutil

```
@@ -0,0 +1,23 @@
+# Copyright (c) 2020 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig MCUBOOT_BOOTUTIL_LIB
+	bool "MCUBoot utility library"
+	help
+	  Enable MCUBoot utility library which implements functions
+	  required by the chain-loaded application and the MCUBoot.
+
+if MCUBOOT_BOOTUTIL_LIB
+
+#hiden option for disabling module-own log configuration
+#while building MCUBoot bootloader
+config DISABLE_MCUBOOT_BOOTUTIL_LIB_OWN_LOG
+	bool
```

Nothing wrong, but generally I think a `MCUBOOT_BOOTUTIL_LIB_LOG=y` is nicer and easier to read.
```suggestion
config MCUBOOT_BOOTUTIL_LIB_LOG
	bool
	default y
```
and then in mcuboot do:
```
config MCUBOOT_BOOTUTIL_LIB_LOG
	bool
	default n
```

this is one of the few places where a `default n` is actually meaningful, and will work when placed in https://github.com/zephyrproject-rtos/mcuboot/blob/master/boot/zephyr/Kconfig before the `Kconfig.zephyr` is sourced:
https://github.com/zephyrproject-rtos/mcuboot/blob/6e3825f1309b34d3f5e5e1be8dd522cfe92e044b/boot/zephyr/Kconfig#L574

Ref: https://docs.zephyrproject.org/latest/guides/kconfig/tips.html#redundant-defaults
> The one case where `default n/default ""` is not redundant is when defining a symbol in multiple locations and wanting to override e.g. a default y on a later definition.

---

reworked